amethyst_ui/src/button/builder.rs

```diff
@@ -1,7 +1,10 @@
-use super::{Anchor, FontAsset, FontHandle, MouseReactive, Stretch, TtfFormat, UiImage, UiText,
-            UiTransform};
-///! A clickable button.
+use super::super::{
```

Comment:
is there a better way than super::supper?

Reply:
Yeah, I'll use an absolute path